### ISSUE TITLE
Non-environment flows lack a category

### ISSUE THREAD

**Issue by bkuczenski** [USEPA/USEEIO#14]
Under the OpenLCA schema, all objects should be categorized. Flows that are elementary have elementary categories; however, many flows are not categorized. See the following two examples, chosen at random from a recent build:

```
{
  "@type": "Flow",
  "@id": "0a69c698-a3b0-3945-884c-0a75e5f4a444",
  "name": "1,4-Dichloro-2-butene",
  "flowType": "ELEMENTARY_FLOW",
  "flowProperties": [
    {
      "referenceFlowProperty": true,
      "conversionFactor": 1.0,
      "flowProperty": {
        "@id": "93a60a56-a3c8-11da-a746-0800200b9a66"
      }
    }
  ],
  "category": {
    "@id": "42c9fee0-d374-30e8-b2c5-32fa901a691e"
  }
}
```

```
{
  "@type": "Flow",
  "@id": "946a7ad3-1042-3ded-8a7f-8368643372ac",
  "name": "Compensation of employees",
  "flowType": "ELEMENTARY_FLOW",
  "flowProperties": [
    {
      "referenceFlowProperty": true,
      "conversionFactor": 1.0,
      "flowProperty": {
        "@id": "3bf53920-157c-4c2f-bddd-7c92c9d35f10"
      }
    }
  ]
}
```

**Comment by bkuczenski** [comment]
It is actually only "a few" flows that are not categorized:


    [('015046c6-bae5-3f2c-9b6e-d2d693275b5a', 'Jobs'),
     ('2e21b0ba-5505-3a0e-a7ec-4cb04c031c70', 'Gross operating surplus'),
     ('946a7ad3-1042-3ded-8a7f-8368643372ac', 'Compensation of employees'),
     ('d9fe7de6-cf17-30f9-8863-d722dd839af7', 'Taxes on production and imports, less subsidies')]

**Comment by WesIngwersen** [comment]
Thanks for raising this @bkuczenski. Because they don't fit the elementary flows definition, they aren't part of the [FEDEFL](https://github.com/USEPA/Federal-LCA-Commons-Elementary-Flow-List). And because they aren't product flows, they don't use the NAICS categories either.  We welcome your suggestions on which categories to use. 

cc @bl-young

**Comment by bl-young** [comment]
When importing into openLCA, I believe these flows get dropped just in the Flows folder:
![image](https://user-images.githubusercontent.com/44471635/155581315-be13db98-433f-495e-8d5a-13caa438c34e.png)

Does this lack of categorization have ramifications elsewhere?

**Comment by bkuczenski** [comment]
The schema does suggest that it is not required. But since the flows are implicitly categorized (tagged, at least) as "ECONOMIC_FLOW", it improves reusability to make that categorization explicit. Note that there are a few other entities for which the omitted categorization "sticks out":
![image](https://user-images.githubusercontent.com/2121413/155593804-8c37f25f-14ee-47ae-947d-c84440453dcc.png)

"Producer price" and "units of currency" both seem to be appropriately "Economic" entities

**Comment by bl-young** [comment]
Yes, I agree it makes sense to embed those flow properties into the correct folder structure. By the way, did you modify them yourself for a different price year?
